mssql 3.3.0 and stream()

bartam opened the pull request:
Added stream() which is useful for larger responses.

misterGF commented:
Thanks for the PR! The changes look great and the test all pass. The one thing I ask is that we follow the library's style format ([standard](http://standardjs.com/rules.html)) and forgo the semicolons.

bartam commented:
I removed the semis and added a try/catch around your bunyan logging.  Bunyan is listed as a devDependency so the adapter wasn't running on a new sails app.
